thoth/prescriptions_refresh/handlers/quay/common.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python3
+# thoth-prescriptions-refresh
+# Copyright(C) 2021 Fridolin Pokorny
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Check security related information in predictable stacks container images."""
```

Comment:
This should describe the common logic right?

Reply:
Adjusted. Thanks! 👍🏻